docker-builds/reference-tests/Dockerfile

```diff
@@ -6,8 +6,6 @@ WORKDIR /demo-test-suite
 RUN pip install -r requirements.txt && pip install fhirclient
 COPY behave.ini.dist /demo-test-suite/behave.ini.dist
 
-VOLUME ["/demo-test-suite/config"]
```

Comment:
This was causing issues where changes to my config files weren't showing up inside the container. I'm not sure why this was present in the first place, so I've removed it.